miniapp/USERGUIDE.md

```diff
@@ -519,6 +534,12 @@ You can also choose to override the default functionality and instead share the
 **API Docs:** [MiniAppMessageBridge.getHostEnvironmentInfo](api/com.rakuten.tech.mobile.miniapp.js/-mini-app-message-bridge/get-host-environment.html)
 
 The default functionality will provide information using `HostEnvironmentInfo` object to Mini App. Also, Host App can send it's environment information by implementing this function.
+    
+### Send Json To Host App
+**API Docs:** [MiniAppMessageBridge.sendJsonToHostApp](api/com.rakuten.tech.mobile.miniapp.js/-mini-app-message-bridge/send-json-to-host-app.html)
+
+The MiniApp is able to send the Universal Bridge. It can be either a message or an event for any purpose according to your need in `json` format. 
```

Comment:
I think this `sendJsonToHostApp` only covers the sending a json string part, we can add event one in the event section
```suggestion
The MiniApp is able to send json format data as a string to Host App.
```

Reply:
Refer to [this page
](https://confluence.rakuten-it.com/confluence/display/ONEAPP/Universal+Bridge)
`We need a generic interface(s) between Hostapp and MiniApp that can communicate freely. Interface(s) can be used for any purpose(By the Host app/MiniApp) according to their need.`

but no worries, change has been made.